### PR TITLE
feat: add priority support(#550)

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

Fix: #550 

method `UpdatePolicy` not support priority yet

### CODE REVIEW

**Comment by hsluoyz** [comment]
@closetool plz fix semantic PR:

image

**Comment by closetool** [comment]
@hsluoyz plz review

**Comment by hsluoyz** [comment]
@wndrr101 please review.

File: enforcer.go [hunk 1]
```diff
@@ -172,6 +172,10 @@ func (e *Enforcer) InitWithModelAndAdapter(m model.Model, adapter persist.Adapte
 		}
 	}
 
+	if err := e.model.SortPoliciesByPriority(); err != nil {
```
Comment (nodece) [inline enforcer.go:175]: Unnecessary code.

**Review by nodece — CHANGES_REQUESTED** [review]
You need to format code after completing the changes.

**Comment by closetool** [comment]
@nodece plz review again

**Pull request merged**